src/main/java/codeu/controller/TestDataServlet.java

```diff
@@ -75,7 +76,10 @@ void setUserStore(UserStore userStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    request.getRequestDispatcher("/WEB-INF/view/testdata.jsp").forward(request, response);
+    User currentUser = (User) request.getSession().getAttribute("user");
```

Comment:
This won't work since in LoginServlet the "user" attribute is set as a string: https://github.com/hsha0/CodeU-Spring2018-Team19/blob/master/src/main/java/codeu/controller/LoginServlet.java#L75

It needs to be changed to store the "User" object instead.